sql/core/src/test/resources/sql-tests/analyzer-results/postgreSQL/window_part3.sql.out

```diff
@@ -336,21 +336,12 @@ org.apache.spark.sql.catalyst.ExtendedAnalysisException
 -- !query
 SELECT rank() OVER (ORDER BY 1), count(*) FROM empsalary GROUP BY 1
 -- !query analysis
-org.apache.spark.sql.catalyst.ExtendedAnalysisException
-{
-  "errorClass" : "UNSUPPORTED_EXPR_FOR_OPERATOR",
-  "sqlState" : "42K0E",
-  "messageParameters" : {
-    "invalidExprSqls" : "\"RANK() OVER (ORDER BY 1 ASC NULLS FIRST ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW)\""
-  },
-  "queryContext" : [ {
-    "objectType" : "",
-    "objectName" : "",
-    "startIndex" : 58,
-    "stopIndex" : 67,
-    "fragment" : "GROUP BY 1"
-  } ]
-}
+Project [RANK() OVER (ORDER BY 1 ASC NULLS FIRST ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW)#x, count(1)#xL]
```

Comment:
why does it show the plan here? The error message doesn't seem improved in this case.